### ISSUE TITLE
Add images: Android Basics with Compose

### ISSUE THREAD

**Issue by sadcoffeee** [google-developer-training/basic-android-kotlin-compose-birthday-card-app#596]
**URL of codelab**
https://developer.android.com/codelabs/basic-android-kotlin-compose-add-images

**In which task and step of the codelab can this issue be found?**
Task 7 step 6-7

**Describe the problem**
The example shows how extracting a string resource replaces it with the getString() function, and this is also what is described in text, but further ahead in the codelab this part is written with the stringResource() function. There is also a Note that you should manually change the getString() functions to stringResource() functions in cases of Android Studio using the wrong one, which stands in start contrast to the examples where getString() is the correct one.

In other words the codelab mixes up getString() and stringResource() and is thus not clear on which is the correct to use.

**Additional information**
![error](https://github.com/google-developer-training/basic-android-kotlin-compose-birthday-card-app/assets/148781287/96cafa9b-dc7c-4152-9597-963fcff67c88)
